dev/buildtool/cloudbuild/containers.yml

```diff
@@ -18,6 +18,8 @@ steps:
             "build",
             "-t", "$_DOCKER_REGISTRY/$_IMAGE_NAME:$TAG_NAME",
             "-t", "$_DOCKER_REGISTRY/$_IMAGE_NAME:$TAG_NAME-slim",
+            "-t", "$_ARTIFACT_REGISTRY/$_IMAGE_NAME:$TAG_NAME",
+            "-t", "$_ARTIFACT_REGISTRY/$_IMAGE_NAME:$TAG_NAME-slim",
```

Comment:
You need to list these under the `images` section or they won't actually get pushed

Reply:
Good catch. Fixed.